tests/pytorch/distributed/run_layer_with_overlap.py

```diff
@@ -107,6 +107,14 @@ def _parse_args(argv=None, namespace=None):
     parser.add_argument(
         "--fp8", action="store_true", default=False, help="Enables the te.fp8_autocast() context."
     )
+    parser.add_argument(
+        "--fp8-recipe",
+        type=str.lower,
+        default="delayed",
+        choices=["delayed", "tensorwise"],
+        help="Which fp8 recipe to use for FP8 tensors in the forward and backward pass",
+        dest="fp8_recipe",
+    )
```

Comment:
We should be able to generalize to MXFP8:
```suggestion
    parser.add_argument(
        "--quantization",
        type=str.lower,
        default="none",
        choices=["none", "fp8_delayed_scaling", "fp8_current_scaling"],
        help="Quantization recipe",
    )
```
See the test I've been using in my MXFP8 branch: https://github.com/timmoon10/TransformerEngine/blob/b8b325bb35a3ac233097a58a8657922f9c9611b5/tests/pytorch/distributed/test_fusible_ops_with_userbuffers.py#L485

Reply:
Have made corresponding changes according to the suggestions.